### ISSUE TITLE
`DocService` gRPC repeated fields are not collapsible

### ISSUE THREAD

**Issue by Dogacel** [line/armeria#4485]
<img width="866" alt="image" src="https://user-images.githubusercontent.com/7023385/196294372-215406ad-ebca-413b-a052-a0dbe90d0815.png">

The `childFieldInfos` are not correctly populated in repeated fields, I think it would be useful to see child fields of the repeated fields instead of needing to click and visit another page.

**Comment by Dogacel** [comment]
@ikhoon regarding the table becoming too large, how about lazy loading child properties? Maybe not in the scope of this change. I haven't seen such big endpoints in my use-case yet.

Also assuming fields in `map` can be a little tricky because maps have two parameters. Maybe it is best to keep it this way. My motivation was using the `debugForm` without needing to navigate through lots of different pages. Beceause I suppose you loose your debug form template when you navigate to a different page.

**Comment by ikhoon** [comment]
> My motivation was using the debugForm without needing to navigate through lots of different pages. 

I agree with your opinion. But I heard @chris-ryan-square manages large protos.
https://line-armeria.slack.com/archives/C1NGPBUH2/p1665528225754379?thread_ts=1665526584.427749&cid=C1NGPBUH2

As you said, lazy loading could be a possible solution.

**Comment by Dogacel** [comment]
@ikhoon Is there a community slack channel for armeria? I thought link you sent was a community channel but I can't really get in 😆
